agent/xds/endpoints.go

```diff
@@ -297,6 +298,16 @@ func makeEndpoint(host string, port int) *envoy_endpoint_v3.LbEndpoint {
 	}
 }
 
+func makePipeEndpoint(path string) *envoy_endpoint_v3.LbEndpoint {
+	return &envoy_endpoint_v3.LbEndpoint{
+		HostIdentifier: &envoy_endpoint_v3.LbEndpoint_Endpoint{
+			Endpoint: &envoy_endpoint_v3.Endpoint{
+				Address: makePipeAddress(path, 0),
```

Comment:
So this is `0` here because it's not creating the socket, just sending on the socket?

Reply:
Yes. No mode is needed here.